public/data/MasterReach1.json

```
@@ -0,0 +1,59 @@
+{
+  "name": "MasterReach1",
+  "type": "personal",
+  "bio": "Web Developer | Ethical Hacker | Open Source Enthusiast",
+  "avatar": "https://github.com/MasterReach1.png",
+  "links": [
+    {
+      "name": "Follow me here on Github!",
```

I recommend capitalizing this `H` in Github, What do you think ⁉

```suggestion
      "name": "Follow me here on GitHub!",
```

---

I think that's a great idea. I have capitalized the H and it looks way better.